### PR TITLE
Participant search improvements

### PR DESCRIPTION
**Pull request by alxdarksage**
BRIDGE-3025

### CODE REVIEW

File: src/main/java/org/sagebionetworks/bridge/hibernate/QueryBuilder.java [hunk 1]
```diff
@@ -16,37 +18,73 @@
...
  */
 class QueryBuilder {
     
     private final List<String> phrases = new ArrayList<>();
     private final Map<String,Object> params = new HashMap<>();
+    private QueryBuilder whereClause;
+    
+    public QueryBuilder startWhere() {
+        whereClause = new QueryBuilder();
+        return whereClause;
```
Comment (alxdarksage) [inline src/main/java/org/sagebionetworks/bridge/hibernate/QueryBuilder.java:35]: Make a WhereQueryBuilder with some of these methods moved over, just so later it doesn't get confusing.

File: src/main/java/org/sagebionetworks/bridge/services/EnrollmentService.java [hunk 2]
```diff
@@ -26,6 +27,7 @@
 import org.sagebionetworks.bridge.exceptions.BadRequestException;
 import org.sagebionetworks.bridge.exceptions.EntityNotFoundException;
 import org.sagebionetworks.bridge.models.PagedResourceList;
+import org.sagebionetworks.bridge.models.ResourceList;
 import org.sagebionetworks.bridge.models.accounts.Account;
 import org.sagebionetworks.bridge.models.accounts.AccountId;
 import org.sagebionetworks.bridge.models.studies.Enrollment;
```
Comment (alxdarksage) [inline src/main/java/org/sagebionetworks/bridge/services/EnrollmentService.java]: Huh? Unused imports?

File: src/main/java/org/sagebionetworks/bridge/hibernate/QueryBuilder.java (hunk 1 above)
Comment (alxdarksage) [reply src/main/java/org/sagebionetworks/bridge/hibernate/QueryBuilder.java:35]: Done

**Review by nategbrown9 — APPROVED** [review]
Looks good.

File: src/main/java/org/sagebionetworks/bridge/services/EnrollmentService.java (hunk 2 above)
Comment (alxdarksage) [reply src/main/java/org/sagebionetworks/bridge/services/EnrollmentService.java]: Yep, removed

**Pull request merged**